README.asciidoc

```diff
@@ -189,9 +189,9 @@ requests for consideration or create an issue with a code change proposal.
 
 === Installing dependencies
 
-On openSUSE one can install the package
-link:https://build.opensuse.org/project/show/devel:openQA[`os-autoinst-devel`] which
-pulls all dependencies.
+On openSUSE one can install the package `os-autoinst-devel` from
+link:https://build.opensuse.org/project/show/devel:openQA[devel:openQA] which
+pulls in all dependencies. See also link:https://open.qa/docs/#_development_version_repository[the openQA docs].
```

Comment:
Not entirely correct. The package is within both openSUSE Leap as well as Tumbleweed and works to develop for the *corresponding version*. For Tumbleweed most likely it will nearly always work because it's recent enough.

Reply:
Neither Leap nor Tumbleweed is mentioned here. It works across versions so I would prefer to leave this generic.

Reply:
Btw installing os-autoinst-devel from default repos on Leap 15.2 is what I originally tried. So I'm against anything that makes it sound like a good idea to use default repos in general.

Reply:
I still think it's a good idea to extend the README but with including that notion. How about just explaining that aspect and letting the reader decide.

Reply:
> one definitely needs additional repos which is not what I want to encourage. The preferred base should be Tumbleweed and only Tumbleweed.

So Tumbleweed should work fine with default repos? In that case I would make it a separate section.

Reply:
What about:
> On openSUSE one can install the package `os-autoinst-devel` which pulls in all dependencies. Under Leap it is recommend to install it from devel:openQA as the distribution-provided packages are usually too old.

Reply:
Not entirely correct (as I stated in https://github.com/os-autoinst/os-autoinst/pull/1858#discussion_r755851692 ) but acceptable.

How about

```suggestion
On openSUSE one can install the package `os-autoinst-devel` which provides all
the dependencies to build and run os-autoinst for the corresponding version of
sources. To build a current version of `os-autoinst` it is recommended to
install `os-autoinst-devel` from
link:https://build.opensuse.org/project/show/devel:openQA[devel:openQA] as the
distribution-provided packages might be too old or miss dependencies. This is
particularly true for openSUSE Leap. Also see
link:https://open.qa/docs/#_development_version_repository[the openQA docs].
```